Src/java/cql-to-elm/src/test/java/org/cqframework/cql/cql2elm/model/ModelImporterTest.java

```diff
@@ -18,15 +19,18 @@
 
 public class ModelImporterTest {
 
-    @Test
+    //@Test
```

Comment:
Agreed, added SkipException (though it's not ideal since it means you have to comment out the rest of the method or Java's "unreachable code" detection kicks in...

Reply:
Oh shoot.  Uh... I think you can do this instead?
```
@Test(enabled = false)
```

Reply:
Tried that and based on a google or two, it should work, but IntelliJ didn't like it, so I just commented out the body of the tests. Not a big deal.